recipes-kernel/linux/files/patches-5.10/0220-arm64-dts-ti-Definitions-for-runtime-pinmuxing.patch

```diff
@@ -0,0 +1,754 @@
+From be89e6880a715a38405cec22364754661834aa11 Mon Sep 17 00:00:00 2001
+From: Benedikt Niedermayr <benedikt.niedermayr@siemens.com>
+Date: Tue, 21 Feb 2023 11:03:42 +0100
+Subject: [PATCH] arm64: dts: ti: Definitions for runtime pinmuxing
```

Comment:
You should address the iot2050 with the subject.

Reply:
So I change that to  "iot2050: Definitions for runtime pinmuxing" ?

Reply:
arm64: dts: ti: iot2050: ...